go/engine/track_token_test.go

```diff
@@ -21,13 +21,9 @@ func doWithSigChainVersions(f func(libkb.SigVersion)) {
 }
 
 func TestTrackTokenIdentify2(t *testing.T) {
-	doWithSigChainVersions(func(sigVersion libkb.SigVersion) {
```

Comment:
I think you should do this one for both...

Reply:
done